### PR TITLE
update clusterset permission in admin/edit

### PR DESCRIPTION
**Pull request by ldpliu**
https://github.com/open-cluster-management/backlog/issues/13014

### CODE REVIEW

File: templates/multiclusterhub/base/rbac/clusterrole-edit-aggregate.yaml [hunk 1]
```diff
@@ -8,8 +8,11 @@ metadata:
   name: open-cluster-management:edit-aggregate
 rules:
 - apiGroups: ["cluster.open-cluster-management.io"]
-  resources: ["managedclusters", "managedclustersets", "managedclusters/status", "clustercurators"]
+  resources: ["managedclusters", "managedclusters/status", "clustercurators"]
```
Comment (qiujian16) [inline templates/multiclusterhub/base/rbac/clusterrole-edit-aggregate.yaml:11]: I think we should remove all cluster scoped resource in admin/editor/view clusterrole. These clusterrole are mainly for ns scoped resource. WDYT  @chaitanyaenr ?

**Comment by zkayyali812** [comment]
/lgtm 

@ldpliu Please remove hold tag to merge when ready and convos are resolved.

**Comment by ldpliu** [comment]
@chaitanyaenr should we remove all cluster scoped resource in admin/editor/view clusterrole, or just keep current change?

**Comment by zkayyali812** [comment]
@ldpliu @qiujian16 @chaitanyaenr 
Is this okay to be merged in, or is it still awaiting further input?

**Comment by qiujian16** [comment]
I am fine to merge it.

**Comment by ldpliu** [comment]
merge this pr, 
@chaitanyaenr if you have comments later, I will submit another pr to handle it.

**Pull request merged**